### DISCUSSION TITLE
KB246332 errors all the time

### DISCUSSION THREAD

**Discussion by thepwrtank18** [LegacyUpdate/LegacyUpdate#79] [Questions & Help]
KB246332 is Service Pack 4 for SQL Server 2005, which was bundled with Visual Studio 2005. It never installs successfully, with the vague error code 1603.

**Comment by tsautier** [comment]
Same here :

<img width="1443" height="770" alt="image" src="https://github.com/user-attachments/assets/b00fbd18-2a5f-4467-951d-f396d5a3788a" />
